src/main/java/org/squiddev/plethora/integration/buildcraft/TileControllable.java

```diff
@@ -0,0 +1,53 @@
+package org.squiddev.plethora.integration.buildcraft;
+
+import buildcraft.api.tiles.IControllable;
+import buildcraft.core.BCCore;
+import dan200.computercraft.api.lua.LuaException;
+import org.squiddev.plethora.api.Injects;
+import org.squiddev.plethora.api.method.wrapper.FromTarget;
+import org.squiddev.plethora.api.method.wrapper.PlethoraMethod;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Injects(BCCore.MODID)
```

Comment:
You don't need `@Injects` if you've only got methods inside the class. `@Injects` is used for fields.